charmcraft/commands/store/__init__.py

```diff
@@ -240,3 +240,49 @@ def run(self, parsed_args):
         table = tabulate(data, headers=headers, tablefmt='plain', numalign='left')
         for line in table.splitlines():
             logger.info(line)
+
+
+class ReleaseCommand(BaseCommand):
+    """Release a charm revision to specific channels."""
+    name = 'release'
+    help_msg = "relase a charm revision to one or more channels"
+
+    def fill_parser(self, parser):
+        """Add own parameters to the general parser."""
+        parser.add_argument(
+            'channels', metavar='channel', nargs='+',
+            help="the channel(s) to release to")
+        parser.add_argument('--name', help="the name of the charm to get revisions")
+        parser.add_argument(
```

Comment:
'snapcraft release --help' says:
```
snapcraft release [OPTIONS] <snap-name> <revision> <channels>
```

Which means you must always supply the positional revision.
Are we intentionally diverging from that? Is it part of the CLI review?

Reply:
We're intentionally diverging from that: with @chipaca we thought it was a good idea to default to "last revision", as it's what you normally release.

Reply:
to be clear, we didn't intentionally diverge on a whim; after talking with snapcraft and ux people, the current snapcraft ux for release is Bad, so we're trying something different